### PR TITLE
Plans: auto open for plans descriptions (refact)

### PR DESCRIPTION
**Pull request by retrofox**
Introduces test mentioned in #6997.
Also this PR tries to reach out the same purpose that #7465. I didn't rebased the `new/plans-hover-tooltip` branch in order to avoid override the code that @artpi  wrote right there.

## Testing

1) Find, activate and deactivate the test from ABTEST option in dev menu.

2) See how the InfoPopover is shown when the user does a `click` or `mouseEnter/mouseLeave` event in function of the test previously selected.

### CODE REVIEW

**Comment by retrofox** [comment]
cc @rralian @lamosty @artpi @gwwar

File: client/lib/abtest/active-tests.js [hunk 1]
```diff
@@ -41,6 +41,15 @@ module.exports = {
 		},
 		defaultVariation: 'singlePurchaseFlow'
 	},
+	plansDescriptions: {
```
Comment (lamosty) [inline client/lib/abtest/active-tests.js]: Can we name this a bit differently? E.g.: plansDescriptionsOnHover. I think `planDescriptions` is too vague. :)

File: client/lib/abtest/active-tests.js [hunk 2]
```diff
@@ -41,6 +41,15 @@ module.exports = {
...
+		datestamp: '20160901',
+		variations: {
+			auto: 50,
```
Comment (lamosty) [inline client/lib/abtest/active-tests.js]: Also, maybe name this "hover"?

File: client/my-sites/plan-features/item.jsx [hunk 3]
```diff
@@ -8,23 +8,33 @@ import React from 'react';
  */
 import Gridicon from 'components/gridicon';
 
+/**
+ * Module variables
```
Comment (lamosty) [inline client/my-sites/plan-features/item.jsx]: I've seen this comment before. Is it a new convention?

**Comment by lamosty** [comment]
Nice work!

When the abtest is set to 'auto' and I hover over the feature item, it should display the description, shouldn't it? That's what #7465 did. But in this PR, you test hovering the tooltip icon, not feature item as @artpi did in #7465.

Or maybe I'm completely mistaken. :D

**Comment by artpi** [comment]
I vote to close this PR to introduce this test:
https://github.com/Automattic/wp-calypso/pull/7463#issuecomment-242028012

**Pull request closed without merging**